package.json

```diff
@@ -103,7 +103,6 @@
     "sourcemap-istanbul-instrumenter-loader": "0.2.0",
     "stripcomment-loader": "0.1.0",
     "style-loader": "0.23.1",
-    "swagger-ui": "3.23.11",
```

Comment:
why remove swagger-ui library?

Reply:
- I think we are unable to access it anymore, since removed the admin page
- The library is throwing annoying errors due to last dependabot fix, so I think just remove it makes no harm

Reply:
i think still can access
@junzai97  have a fix in his branch for this (if not mistake)